integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -131,6 +131,7 @@ def generate_dest_data(spark):
 @pytest.mark.parametrize("use_cdf", [True, False], ids=idfn)
 @pytest.mark.parametrize("partition_columns", [None, ["a"]], ids=idfn)
 @pytest.mark.skipif(is_before_spark_320(), reason="Delta Lake writes are not supported before Spark 3.2.x")
+@pytest.mark.skipif(is_databricks122_or_later(), reason="https://github.com/NVIDIA/spark-rapids/issues/8423")
```

Comment:
Technically we want all these to be xfail'd tests rather than skipped.  We xfail tests that are failing due to bugs and should work at some point in the future when those bugs are fixed.  We skip tests that we never expect to work.  Definitely not must-fix.

Reply:
Makes sense.